### PR TITLE
Add kubebuilder shortName tag

### PR DESCRIPTION
**Pull request by pigletfly**
**What type of PR is this?**
/kind cleanup 

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes https://github.com/karmada-io/karmada/issues/348

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: pkg/apis/work/v1alpha1/binding_types.go [hunk 1]
```diff
@@ -97,6 +98,7 @@ type ResourceBindingList struct {
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
 // +kubebuilder:resource:scope="Cluster"
 // +kubebuilder:subresource:status
+// +kubebuilder:resource:shortName=crb
```
Comment (RainbowMango) [inline pkg/apis/work/v1alpha1/binding_types.go:101]: I‘d prefer to put `shortName` and `scope` together, like 
```
// +kubebuilder:resource:scope="Cluster",shortName=crb
```

File: artifacts/deploy/work.karmada.io_clusterresourcebindings.yaml [hunk 2]
```diff
@@ -13,8 +13,10 @@ spec:
     kind: ClusterResourceBinding
     listKind: ClusterResourceBindingList
     plural: clusterresourcebindings
+    shortNames:
+    - crb
     singular: clusterresourcebinding
-  scope: Cluster
+  scope: Namespaced
```
Comment (RainbowMango) [inline artifacts/deploy/work.karmada.io_clusterresourcebindings.yaml:19]: Wow, unexpected change, and caused by https://github.com/karmada-io/karmada/pull/376/files#r639349073

File: pkg/apis/work/v1alpha1/binding_types.go (hunk 1 above)
Comment (RainbowMango) [reply pkg/apis/work/v1alpha1/binding_types.go:101]: That also explains the CI failure.

Comment (pigletfly) [reply pkg/apis/work/v1alpha1/binding_types.go]: done

**Comment by RainbowMango** [comment]
LGTM.  Thanks for doing this.
As we said at the community meeting, just hold for a while(until this week would be enough, I think) to see if other people have any comments.

**Comment by RainbowMango** [comment]
Ok. No other comments, let's push this forward.

**Comment by RainbowMango** [comment]
Great thanks @pigletfly .

**Pull request merged**